nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py

```diff
@@ -0,0 +1,267 @@
+       be used as waveform data.
+
+ Steps:
+    1. Create a task.
+    2. Create an Analog Output Current channel.
+    3. Define the update Rate for the Current generation.
+       Additionally, define the sample mode to be continuous.
+    4. Write the waveform to the output buffer.
+    5. Call the Start function.
+    6. Wait until the user presses the Stop button.
+    7. Call the Clear Task function to clear the Task.
+    8. Display an error if any.
+
+ I/O Connections Overview:
+    Make sure your signal output terminal matches the Physical
+    Channel I/O Control. For further connection information, refer
+    to your hardware reference manual
+"""
+
+
+import nidaqmx
+import math
+from nidaqmx.constants import AcquisitionType
+import nidaqmx.errors as DAQmxError
+import time
+import sys
+import os
+import numpy as np
+
+
+# Input Parameters used for DAQmx channel configuration
+phys_channel_names = ['Current_Output/ao0', 'Current_Output/ao1']
+range_max       = 0.02
+range_min       = 0.00
+frequency       = 100
+amplitude       = 0.01
+offset          = amplitude
+sampsPerBuffer  = 250
+cyclesPerBuffer = 5
+sampsPerCycle = sampsPerBuffer / cyclesPerBuffer
+
+waveform_types = ['Triangle', 'Square', 'Sawtooth', 'Sine']
+
+#Global variables 
+resultingFrequency = 0
+desiredSampClkRate = 0
+dutyCycle          = 50.0
+Phase              = 0
+data               = []
+
+#------------------------------------
+# Auxiliar Functions
+#------------------------------------
+
+def exit_application():
+  """
+  This function is used to terminate the application in case of an error
+  or in case of keyboard interrupt
+  """
+  print ("Application Ended")
+  try:
+    sys.exit(0)
+  except SystemExit:
+    os._exit(0)
+
+def print_error_info(error):
+  print("DAQmx error", error.error_code, "occurred")
+  print("Cause: ", error.error_type)
+
+def ConfigureSampleClkTiming(desiredFreq, sampsPerbuff, cyclesPerBuff, DAQmxTask,sampsPerCycle):
+  """
+  This function helps to set the Sample Rate for the On-Demand task, based on the samples per buffer,
+  Cycles per buffer and desired Waveform Frequency. It also sets the Acquisition Mode (Finite / Continuous)
+  and the samples per channel.
+
+  Inputs:
+    desiredFreq : Is the desired Frequency for the selected Waveform [Hz]
+    sampsPerBuff: Number of samples to write in the output buffer
+    cyclesPerBuff: Number of cycles within a buffer write operation
+    sampsPerCycle: Number of samples to write on a single cycle
+
+  Outputs:
+    desiredSampClkRate: Is the calculated sample rate to be used at the DAQmx timing configuration.
+    resultingFrequency: I DON'T KNOW WHAT THIS WAS USED FOR 
+  """
+  sampsPerCycle = sampsPerbuff / cyclesPerBuff
+  desiredSampClkRate = desiredFreq * sampsPerbuff / cyclesPerBuff
+  resultingFrequency = desiredSampClkRate / sampsPerCycle
+
+  try:
+    DAQmxTask.timing.cfg_samp_clk_timing(desiredSampClkRate, sample_mode = AcquisitionType.CONTINUOUS)
```

Comment:
`DAQmxTask` is a confusing name. I thought this was calling a static function on a class. Can you rename the variable `task` to match the rest of the program?